app/retail/emails.py

```diff
@@ -972,45 +972,45 @@ def render_new_bounty_roundup(to_email):
 </p>
 '''
     highlights = [{
-        'who': 'IgorShadurin',
+        'who': 'dcd018',
         'who_link': True,
-        'what': 'Provable Emails, thanks to IgorShadurin and josh-richardson',
-        'link': 'https://gitcoin.co/issue/ArweaveTeam/Bounties/5/3164',
+        'what': 'Clone form, fixed!',
+        'link': 'https://gitcoin.co/issue/gitcoinco/code_fund_ads/572/3247',
         'link_copy': 'View more',
     }, {
-        'who': 'sanchaymittal',
+        'who': 'enieber',
         'who_link': True,
-        'what': 'Docs are complete!',
-        'link': 'https://gitcoin.co/issue/MrElliwood/audio-router/1/3213',
+        'what': 'README: Updated!',
+        'link': 'https://gitcoin.co/issue/sigillabs/mobidex/236/3166',
         'link_copy': 'View more',
     }, {
-        'who': 'cpurta',
+        'who': 'gutsal-arsen',
         'who_link': True,
-        'what': 'Raiden Hackathon Completion!',
-        'link': 'https://gitcoin.co/issue/raiden-network/hackathons/3/3170',
+        'what': 'Safe Keyboard, Implemented',
+        'link': 'https://gitcoin.co/issue/polkawallet-io/polkawallet-RN/104/2944',
         'link_copy': 'View more',
     }, ]
 
     sponsor = {
-        'name': 'Solana',
-        'title': 'Solana is the most performant blockchain in the world with speeds over 50,000 TPS (while being decentralized and secure).',
+        'name': 'Allinfra',
+        'title': 'The Future of Infrastructure Finance',
         'image_url': 'https://s3.us-west-2.amazonaws.com/gitcoin-static/jDSk7ZTfpY19PWdwwsk8puNd.png',
-        'link': 'http://bit.ly/TourDeSOL',
-        'cta': 'Sign Up for Tour de SOL',
+        'link': 'http://bit.ly/Allinfra',
+        'cta': 'Register today',
         'body': [
-            'We just announced Tour de SOL, or our incentivized testnet event. Are you a validator? Earn token and race against the best in the world. Deadline is July 21st.'
+            'Bringing access, choice, and liquidity to unlisted infrastructure assets using the power of Ethereum'
         ]
     }
 
     bounties_spec = [{
-        'url': 'https://github.com/RibbonBlockchain/IncentivesMVP/issues/19',
-        'primer': 'Create Initial Pilot phase for Ribbon blockchain',
+        'url': 'https://github.com/gitcoinco/web/issues/4744',
+        'primer': 'Allow Ability to Resurface Bounties to the Top When Expiration Date is Extended',
     }, {
-        'url': 'https://github.com/RibbonBlockchain/IncentivesMVP/issues/8',
-        'primer': 'Enable camera on front end for capture passport on patient and practitioner onboarding',
+        'url': 'https://github.com/SpeckleOS/speckle-browser-extension/issues/89',
+        'primer': 'Global Settings (Change Colour Theme and Log Out)',
     }, {
-        'url': 'https://github.com/centrifuge/precise-proofs/issues/88',
-        'primer': 'Optimize the fixed height tree implementation',
+        'url': 'https://gitcoin.co/issue/ironcoinext/IronCoin/8/3253',
```

Comment:
You have to use github.com links for this, not gitcoin.co links @vs77bb @ryan-shea